frontend/javascripts/banners.tsx

```diff
@@ -144,3 +152,101 @@ export function MaintenanceBanner() {
 
   return null;
 }
+
+export function UpgradeVersionBanner() {
+  const white = "var(--ant-color-text-primary)";
+  const blue = "var(--ant-color-primary)";
+  const UPGRADE_BANNER_STYLE: React.CSSProperties = {
+    position: "absolute",
+    top: 0,
+    left: 0,
+    height: constants.BANNER_HEIGHT,
+    textAlign: "center",
+    backgroundColor: blue,
+    color: white,
+    fontSize: "medium",
+    minWidth: "fit-content",
+  };
+  const currentDate = dayjs();
+
+  const activeUser = useSelector((state: OxalisState) => state.activeUser);
+  const isVersionOutdated = useFetch(
+    async () => {
+      Utils.sleep(INITIAL_DELAY);
```

Comment:
```suggestion
      await Utils.sleep(INITIAL_DELAY);
```

Reply:
also you can add a `if (!activeUser) return false` at the top of this hook (don't forget to add activeUser to the dependencies). this saves one request for not-logged-in-users.